Reflection/ReflectionProperty.php

```diff
@@ -236,7 +236,7 @@ public function setAccessible($accessible)
      * Gets property type
      *
      * @link https://php.net/manual/en/reflectionproperty.gettype.php
-     * @return ReflectionType|null Returns a {@see ReflectionType} if the
+     * @return ReflectionNamedType|null Returns a {@see ReflectionNamedType} if the
```

Comment:
in real life, it always returns ReflectionNamedType: https://3v4l.org/Y2Jqo

Reply:
While it is safe to say that it will always be a ReflectionNamedType in PHP 7.4, it can also be a ReflectionUnionType in PHP 8 if union types are used. Please see https://3v4l.org/3X0Rp.

So the correct stub type is the common parent, ReflectionType or LanguageLevelTypeAware should be used again. Please update this change

Reply:
@isfedorov 
great note, thanks a lot! Fixed by https://github.com/JetBrains/phpstorm-stubs/pull/1039/commits/4807a64f80bdb9b0539dc55e9c74702aa1225682

It's still not clear to me on should I update PHPDoc blocks for these methods: they uses `@return ReflectionType|null Returns a {@see ReflectionType}`. But since we can't differentiate PHP dock blocks for different PHP versions, it’s not 100% valid to use union types like `@return ReflectionType|ReflectionNamedType|ReflectionUnionType|null`. Alternatively, we can re-declare these methods and use `PhpStormStubsElementAvailable` attribute (as we do, for example, with `bcscale` function). What do you think about it?

Reply:
Yes, I think `PhpStormStubsElementAvailable` can be used for this.